src/test/migrations/steps/20240502105432-step-delete-node-total-land-area.ts

```diff
@@ -0,0 +1,23 @@
+import { Promises } from 'utils/promises'
+
+import { AssessmentController } from 'server/controller/assessment'
+import { BaseProtocol, Schemas } from 'server/db'
+
+export default async (client: BaseProtocol) => {
+  const assessments = await AssessmentController.getAll({}, client)
+  await Promises.each(assessments, async (assessment) => {
+    await Promises.each(assessment.cycles, async (cycle) => {
+      const schemaAssessment = Schemas.getName(assessment)
+      const schemaCycle = Schemas.getNameCycle(assessment, cycle)
+
+      await client.none(`
```

Comment:
It is already using node_ext:

![image](https://github.com/openforis/fra-platform/assets/5508251/7dfdc3f7-c777-474d-a21d-072e958e5d7f)

For countries, which table are you referring to?

Reply:
Great, totalLandArea is not included in extentOfForest

I will fix this in upcoming PR to avoid making this PR too big

I am still trying to solve why the data doesn't show up correctly anymore for regions